### ISSUE TITLE
Looking for detail LastfM dataset

### ISSUE THREAD

**Issue by ZsZsZs25** [gusye1234/LightGCN-PyTorch#11]
Could you provide the detail dataset or link of the train and test samples like [hetrec2011-lastfm-2k data](https://files.grouplens.org/datasets/hetrec2011/hetrec2011-lastfm-2k.zip) in [grouplens](https://grouplens.org/datasets/hetrec-2011/), because I need more information(image, text and so on) about the users or artists in train and test set.
But after comparison, I found the data in all_data.csv is not consistent with [hetrec2011-lastfm-2k data](https://files.grouplens.org/datasets/hetrec2011/hetrec2011-lastfm-2k.zip). If some stupid mistakes I had made, please point out! Thanks a lot!!!

**Comment by gusye1234** [comment]
Hi, thanks for your kindly question.
However, I 'm not sure the lastfm dataset should be consistent with your dataset. For more information about this dataset, you can refer to the paper: [samwalker](https://dl.acm.org/doi/10.1145/3308558.3313582), cause the dataset was provided by the authers to set a simple baseline. 
We posted this dataset meant for those users that didn't have a GPU to train a LightGCN, and I think maybe you can extract the files lightGCN needs from your dataset.

**Comment by ZsZsZs25** [comment]
> Hi, thanks for your kindly question.
> However, I 'm not sure the lastfm dataset should be consistent with your dataset. For more information about this dataset, you can refer to the paper: [samwalker](https://dl.acm.org/doi/10.1145/3308558.3313582), cause the dataset was provided by the authers to set a simple baseline.
> We posted this dataset meant for those users that didn't have a GPU to train a LightGCN, and I think maybe you can extract the files lightGCN needs from your dataset.

Thanks a lot for your reply!!
I  will check it.
